<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!-- We resume the logic in empty_world.launch, changing the name of the world to be launched -->
  <include file="$(find baxter_gazebo)/launch/baxter_world.launch"/>

  <!-- Start the find blocks node -->
  <!--
  <node pkg="block_mover" type="block_finder.py" name="block_finder" />
  -->

  <!-- Start the find blocks node -->
  <node pkg="block_mover" type="block_picker.py" name="block_picker" />

</launch>
